scene/resources/text_paragraph.cpp

```diff
@@ -831,6 +832,7 @@ TextParagraph::TextParagraph(const String &p_text, const Ref<Font> &p_fonts, int
 	spacing_top = p_fonts->get_spacing(Font::SPACING_TOP);
 	spacing_bottom = p_fonts->get_spacing(Font::SPACING_BOTTOM);
 	width = p_width;
+	lines_dirty = true;
```

Comment:
Is it necessary, `lines_dirty` should be already set to `true` by default.

Reply:
Oops, didn't notice it. Fixed.